StratisCore.UI/src/app/wallet/receive/receive.component.ts

```diff
@@ -28,9 +28,11 @@ export class ReceiveComponent {
   public pageNumberUsed: number = 1;
   public pageNumberUnused: number = 1;
   public pageNumberChange: number = 1;
+  public sidechainEnabled: Boolean;
```

Comment:
I think the best practive is not to use Boolean (the one with capital B), because
```
var falseObj = new Boolean(false);
if (falseObj) {
    console.log('our intent may not be to get here');
}
```